sphinxext/remoteliteralinclude.py

```diff
@@ -62,14 +63,13 @@ def read_file(self, url, location=None):
         # try:
         # with codecs.open(url, 'r', self.encoding, errors='strict') as f:  # type: ignore  # NOQA
         #     text = f.read()  # type: unicode
-        response = requests.get(url)
+        response = requests.get(
+            url, headers={"User-Agent": "sphinxext-remoteliteralinclude"}
```

Comment:
Generally you're supposed to put a version here too.

Reply:
My copy and paste python skills failed me trying to get the version from setup.py from perusing stackexchange using importlib

Reply:
User agents for bots are a mess anyway, trying to evade blocks. If it works, it works